test/data/openapi3/normal.yml

```diff
@@ -0,0 +1,539 @@
+openapi: 3.0.0
```

Comment:
Quite minor, but would you mind changing these to use the `.yaml` extension instead of `.yml`? This is [the extension recommended by the project](https://yaml.org/faq.html).